### PR TITLE
Add menu documentation

### PR DESCRIPTION
**Pull request by khangmach**
- Add documentation for dropdown menus
- Add documentation for filter menus

### CODE REVIEW

**Review by jtamiace — APPROVED** [review]
documentation looks good

File: docs/pages/menus/index.vue [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+
+      <p>Menus allow users to make a selection from a list of options. We use two types of menus:</p>
```
Comment (nucleogenesis) [inline docs/pages/menus/index.vue:7]: RE: "We use two types of menus:" 

This colon type of text followed by big headings struck me as an incomplete sentence. Perhaps this should say:

"We use two types of menus: *Dropdown* and *Filter*."

File: docs/pages/menus/index.vue [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+      <h3>
+        Dropdown menu
+      </h3>
+      <p>
+        Allow users to discover and access complex actions. They can be generated from buttons, icons, and in some cases by right-clicking a location
```
Comment (nucleogenesis) [inline docs/pages/menus/index.vue:13]: Here and a few other places, there are `.` periods missing at the end of paragraph sentences.

File: docs/pages/menus/index.vue [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      </p>
+
+      <DocsShow>
+        <img class="modal-img" src="./dropdown-menu.png">
+      </DocsShow>
+
+      <h3>Filter menu</h3>
+      <p>Filter menus allow users to filter information by selecting from a long list of options. They can be generated by clicking a text field</p>
```
Comment (nucleogenesis) [inline docs/pages/menus/index.vue:21]: `.`

**Review by nucleogenesis — APPROVED** [review]
Super minor semantic and syntax suggestions, otherwise everything looks great @khangmach

**Review by nucleogenesis — APPROVED** [review]
Great work @khangmach

**Pull request merged**